certified-connectors/DocumentTranslator/apiDefinition.swagger.json

```diff
@@ -0,0 +1,907 @@
+                    "total": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "total"
+                    },
+                    "failed": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "failed"
+                    },
+                    "success": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "success"
+                    },
+                    "inProgress": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "inProgress"
+                    },
+                    "notYetStarted": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "notYetStarted"
+                    },
+                    "cancelled": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "cancelled"
+                    },
+                    "totalCharacterCharged": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "totalCharacterCharged"
+                    }
+                  },
+                  "description": "summary"
+                }
+              }
+            }
+          },
+          "default": {
+            "description": "Operation Failed."
+          }
+        },
+        "summary": "Get translation status",
+        "description": "Gets a summary of the status for a specific document translation request.",
+        "operationId": "GetTranslationStatus",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The operation ID",
+            "x-ms-summary": "Operation ID",
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "Ocp-Apim-Subscription-Key",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "The value is the Azure key for your Translator or Cognitive Services resource.",
+            "x-ms-summary": "Azure subscription key",
+            "x-ms-visibility": "important"
+          }
+        ]
+      },
+      "delete": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "description": "id"
+                },
+                "createdDateTimeUtc": {
+                  "type": "string",
+                  "description": "createdDateTimeUtc"

```

Comment:
@HamshavathiM description values in this casing are considered non-translatable. Do you want to keep them this way?
If not, please make sure that descriptions look like valid sentences. More information about descriptions can be found in [connector string guidance](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance).

Reply:
Thanks for the review. Resolved